### DISCUSSION TITLE
Any way to use Midori AI Manager with glibc 2.28 ?

### DISCUSSION THREAD

**Discussion by netnut404** [mudler/LocalAI#1762] [General]
I have a machine that must remain for a time glibc 2.28  ( Rocklylinux 8.x ) to be compatible with some other software pre-existing  on the machine.  With that said, is there any way to use the AI Manager with that?  (manually seems to work  but I really want the manger)

Thanks for your time.

Net Nut

**Comment by lunamidori5** [comment]
I did not see that as it is on the LocalAI Github - @netnut404 I would love to support you, can you help give me more info?

**Reply by lunamidori5** [reply]
Open a Issue on the managers github - https://github.com/lunamidori5/Midori-AI
